blog: meet our contributors APAC India region (#279)
kubernetes/contributor-site: content/en/blog/2022/2022-01-06-meet-our-contributors-APAC-India-region-01.md

---
layout: blog
title: "Meet Our Contributors - APAC (India region)"
date: 2022-01-06

> slug: meet-our-contributors-APAC-India-01

Reviewer: ```suggestion
slug: meet-our-contributors-india
```

Author: acknowledged.

---

**Authors & Interviewers:** [Anubhav Vardhan](https://github.com/anubha-v-ardhan), [Atharva Shinde](https://github.com/Atharva-Shinde), [Avinesh Tripathi](https://github.com/AvineshTripathi), [Debabrata Panigrahi](https://github.com/Debanitrkl), [Kunal Verma](https://github.com/verma-kunal), [Pranshu Srivastava](https://github.com/PranshuSrivastava), [Pritish Samal](https://github.com/CIPHERTron), [Purneswar Prasad](https://github.com/PurneswarPrasad), [Vedant Kakde](https://github.com/vedant-kakde)

> **Editor:** [Priyanka Saggu](psaggu.com)

Reviewer: The hyperlink here is not a valid URI.

Author: acknowledged.

---

> Good day, everyone :wave: 

Reviewer: ```suggestion
Good day, everyone! 👋
```

Author: acknowledged.

Welcome to the first episode of the APAC edition of the "Meet Our Contributors" blog post series.

> In this post, we'll meet five amazing folks from the India region who have been actively contributing to the Upstream Kubernetes projects in a variety of ways, as well as being the leaders or maintainers of numerous community initiatives.

Reviewer: How about:
```suggestion
In this post, we'll introduce you to five amazing folks from the India region who have been actively contributing to the upstream Kubernetes projects in a variety of ways, as well as being the leaders or maintainers of numerous community initiatives.
```

Author: acknowledged.

> :dizzy: *Let's get started without further ado ...*   

Reviewer: ```suggestion
💫 *Let's get started, so without further ado…*
```

Author: acknowledged.

## [Arsh Sharma](https://github.com/RinkiyaKeDad)

Arsh was recently employed with VMware as a Member of Technical Staff. As a new contributor, he realised that 1:1 mentorship opportunities were quite beneficial in getting him started with the upstream project.

> He is presently a CI Signal shadow on the Kubernetes 1.23 release team. He is also contributing to the sig-testing and sig-docs projects, as well as the [cert-manager](https://github.com/cert-manager/infrastructure) tools development work that is being done under the sig-architecture.

Reviewer: ```suggestion
He is presently a CI Signal shadow on the Kubernetes 1.23 release team. He is also contributing to SIG Testing and SIG Docs, as well as to the [cert-manager](https://github.com/cert-manager/infrastructure) tools development work that is being done under the aegis of SIG Architecture.
```

Author: acknowledged.

To the newcomers, Arsh helps plan their early contributions sustainably.

> > "I would encourage folks to contribute in a way that's sustainable. What I mean by that is that it's easy to be very enthusiastic early on and take up more stuff than one can actually handle. This can often lead to burnout in later stages. It's much more sustainable to work on things iteratively."

Reviewer: No need for quote marks IMO.
```suggestion
> _I would encourage folks to contribute in a way that's sustainable. What I mean by that
> is that it's easy to be very enthusiastic early on and take up more stuff than one can
> actually handle. This can often lead to burnout in later stages. It's much more sustainable
> to work on things iteratively._
```

Author: acknowledged.

## [Kunal Kushwaha](https://github.com/kunal-kushwaha)

> Kunal Kushwaha is a core member of the Kubernetes marketing council. He is also a CNCF ambassador and one of the founders of the "CNCF Students" program. He also served as a Communications role shadow during the 1.22 release cycle.

Reviewer: ```suggestion
Kunal Kushwaha is a core member of the Kubernetes marketing council. He is also a CNCF ambassador and one of the founders of the [CNCF Students Program](https://community.cncf.io/cloud-native-students/). He also served as a Communications role shadow during the 1.22 release cycle.
```

Author: acknowledged.

> Kunal Kushwaha is a core member of the Kubernetes marketing council. He is also a CNCF ambassador and one of the founders of the "CNCF Students" program. He also served as a Communications role shadow during the 1.22 release cycle.

Reviewer 2: I'm not sure who the "Kubernetes marketing council" is?  Is this supposed to be Contributor Comms, or some other team?

Reviewer 3: Yeah this is the contributor comms team, it was recently renamed

Reviewer 3: https://github.com/kubernetes/community/blob/master/communication/marketing-team/role-handbooks/council.md

Reviewer 4: Yeah, this was a change made by @mbbroberg (with all our approvals). I will admit, the marketing team has several names at this point. Maybe more consistency throughout the multiple envs would help.

At the end of his first year, Kunal began contributing to the [fabric8io kubernetes-client](https://github.com/fabric8io/kubernetes-client) project. He was then selected to work on the same project as part of Google Summer of Code. Kunal mentored people on the same project, first through Google Summer of Code then through Google Code-in. 

As an open-source enthusiast, he believes that diverse participation in the community is beneficial since it introduces new perspectives and opinions and respect for one's peers. He has worked on various open-source projects, and his participation in communities has considerably assisted his development as a developer.

> > “I believe if you find yourself in a place where you do not know much about the project, that's a good thing because now you can learn while contributing and the community is there to help you. It has helped me a lot in gaining skills, meeting people from around the world and also helping them. You can learn on the go, you don't have to be an expert. Make sure to also check out no code contributions because being a beginner is a skill and you can bring new perspectives to the organisation.”

Reviewer: No need for quote marks IMO.
```suggestion
> _I believe if you find yourself in a place where you do not know much about the
> project, that's a good thing because now you can learn while contributing and the
> community is there to help you. It has helped me a lot in gaining skills, meeting
> people from around the world and also helping them. You can learn on the go,
> you don't have to be an expert. Make sure to also check out no code contributions
> because being a beginner is a skill and you can bring new perspectives to the
> organisation._
```

## [Madhav Jivarajani](https://github.com/MadhavJivrajani)

> Madhav Jivarajani works on the VMware Upstream Kubernetes stability team. He began contributing to the Kubernetes project in January 2021 and has since made significant contributions to several areas of work under sig-architecture, sig-api-machinary, and sig-contribex.

Reviewer: ```suggestion
Madhav Jivarajani works on the VMware Upstream Kubernetes stability team. He began contributing to the Kubernetes project in January 2021 and has since made significant contributions to several areas of work under SIG Architecture, SIG API Machinery, and SIG ContribEx (contributor experience).
```

Author: acknowledged.

Among several significant contributions are his recent efforts toward the Archival of [design proposals](https://github.com/kubernetes/community/issues/6055), refactoring the ["groups" codebase](https://github.com/kubernetes/k8s.io/pull/2713) under k8s-infra repository to make it mockable and testable, and improving the functionality of the [GitHub k8s bot](https://github.com/kubernetes/test-infra/issues/23129).

In addition to his technical efforts, Madhav oversees many projects aimed at assisting new contributors. He organises bi-weekly "KEP reading club" sessions to help newcomers understand the process of adding new features, deprecating old ones, and making other key changes to the upstream project. He has also worked on developing [Katacoda scenarios](https://github.com/kubernetes-sigs/contributor-katacoda) to assist new contributors to become acquainted with the process of contributing to k/k. In addition to his current efforts to meet with community members every week, he has organised several [new contributors workshops (NCW)](https://www.youtube.com/watch?v=FgsXbHBRYIc).

> > “I initially did not know much about Kubernetes. I joined because the community was super friendly. But what made me stay was not just the people, but the project itself. My solution to not feeling overwhelmed in the community was to gain as much context and knowledge into the topics that I was interested in and were being discussed. And as a result I continued to dig deeper into kubernetes and the design of it. I am a systems nut & thus kubernetes was an absolute goldmine for me.”

Reviewer: ```suggestion
> _I initially did not know much about Kubernetes. I joined because the community was
> super friendly. But what made me stay was not just the people, but the project itself.
> My solution to not feeling overwhelmed in the community was to gain as much context
> and knowledge into the topics that I was interested in and were being discussed. And
> as a result I continued to dig deeper into Kubernetes and the design of it.
> I am a systems nut & thus Kubernetes was an absolute goldmine for me._
```

## [Rajas Kakodkar](https://github.com/rajaskakodkar)

Rajas Kakodkar currently works at VMware as a Member of Technical Staff. He has been engaged in many aspects of the upstream Kubernetes project since 2019.

> He is now a key contributor to the sig-testing project. He is also active in the sig-network community. Lately, Rajas has contributed significantly to the [netpol](https://docs.google.com/document/d/1AtWQy2fNa4qXRag9cCp5_HsefD7bxKe3ea2RPn8jnSs/edit#heading=h.ajvcztp6cza) and [kpng](https://github.com/kubernetes-sigs/kpng) sub-projects.

Reviewer: ```suggestion
He is now a key contributor to the Testing special interest group. He is also active in the SIG Network community. Lately, Rajas has contributed significantly to the [NetworkPolicy++](https://docs.google.com/document/d/1AtWQy2fNa4qXRag9cCp5_HsefD7bxKe3ea2RPn8jnSs/) and [`kpng`](https://github.com/kubernetes-sigs/kpng) sub-projects.
```

Author: acknowledged.

One of the first challenges he ran across was that he was in a different time zone than the upstream project's regular meeting hours. However, async interactions on community forums progressively corrected that problem.

> > "I enjoy contributing to Kubernetes not just because I get to work on cutting edge tech but more importantly because I get to work with awesome people and help in solving real world problems.”

Reviewer: No need for quote marks IMO
```suggestion
> _I enjoy contributing to Kubernetes not just because I get to work on
> cutting edge tech but more importantly because I get to work with
> awesome people and help in solving real world problems._
```

## [Rajula Vineeth Reddy](https://github.com/rajula96reddy)

> Rajula Vineeth Reddy, a Junior Engineer at CERN, is a member of the Marketing Council team under sig-contribex . He also served as a release shadow for sig-release during the 1.22 and 1.23 Kubernetes release cycles.

Reviewer: ```suggestion
Rajula Vineeth Reddy, a Junior Engineer at CERN, is a member of the Marketing Council team under SIG ContribEx. He also served as a release shadow for sig-release during the 1.22 and 1.23 Kubernetes release cycles.
```

> He started looking at the Kubernetes project as part of a university project with the help of one of his professors. Over time, he spent a significant amount of time reading the project's documentation, slack discussions, GitHub issues, and blogs, which helped him better grasp the Kubernetes project and piqued his interest in contributing upstream. One of his key contributions was his assistance with automation in the SIG-ContribEx Upstream Marketing subproject.

Reviewer: ```suggestion
He started looking at the Kubernetes project as part of a university project with the help of one of his professors. Over time, he spent a significant amount of time reading the project's documentation, Slack discussions, GitHub issues, and blogs, which helped him better grasp the Kubernetes project and piqued his interest in contributing upstream. One of his key contributions was his assistance with automation in the SIG ContribEx Upstream Marketing subproject.
```

Author: acknowledged.

According to Rajula, attending project meetings and shadowing various project roles are vital for learning about the community.

> "I find the community very helpful and it's always *"you get back as much as you contribute"*. The more involved you are, the more you will understand, get to learn and contribute new things.
> 

> > The first step to *"come forward and start"* is hard. But it's all gonna be smooth after that. Just take that jump"

Reviewer: No need for outer quote marks IMO.

```suggestion
> _I find the community very helpful and it's always_
> “you get back as much as you contribute”.
> _The more involved you are, the more you will understand, get to learn and
> contribute new things._
> 
> _The first step to_ “come forward and start” _is hard. But it's all gonna be
> smooth after that. Just take that jump._

---

If you have any recommendations/suggestions for who we should interview next, please let us know in #sig-contribex. We're thrilled to have other folks assisting us in reaching out to even more wonderful individuals of the community. Your suggestions would be much appreciated.

> We'll see you all in the next one. Everyone, till then, have a happy contributing! :wave: 

Reviewer: ```suggestion
We'll see you all in the next one. Everyone, till then, have a happy contributing! 👋
```

Author: acknowledged.
